packages/web3-eth-contract/src/contract.ts

```diff
@@ -427,127 +363,299 @@ export class Contract<Abi extends ContractAbi>
 			set: (value: ContractAbi) => this._parseAndSetJsonInterface(value, returnDataFormat),
 			get: () => this._jsonInterface,
 		});
+
+		this.setConfig(Contract.contractWeb3Config);
 	}
 
+	/**
+	 * Can be used to set {@link Contract.defaultAccount} for all contracts.
+	 */
+	public static get defaultAccount() {
+		return (this.constructor as typeof Contract).defaultAccount;
+	}
+
+	public static set defaultAccount(value: Address | undefined) {
+		(this.constructor as typeof Contract).defaultAccount = value;
```

Comment:
Why property is being set at class level in static method if we have a static obj holding these properties as well: contractWeb3Config

Reply:
The `contractWeb3Config`was meant to be `private`, just to keep tracked of updated properties and be passed in `setConfig()` in `constructor` during initialization.
I kept the distinct static properties, to keep the same "interface" for the class. But, yes we could keep only `contractWeb3Config`.

Reply:
If we decide to keep defaults at contracts level shared, I think we should only: 

>  But, yes we could keep only contractWeb3Config.